### PR TITLE
Update contoso-migration-refactor-web-app-sql.md

### PR DESCRIPTION
**Pull request by dellis2121**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@dellis2121 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Hi @dellis2121 - This PR was opened in the public repository. PMs should work in the private repo, per the contributor guide. Would you open future PRs in the private repo? Thanks!

**Comment by ktoliver** [comment]
@BrianBlanchard - Would you review the proposed revisions to your article? Thanks.

File: docs/migrate/azure-best-practices/contoso-migration-refactor-web-app-sql.md [hunk 1]
```diff
@@ -107,14 +108,25 @@ Here's how Contoso will run the migration:
 
 > [!div class="checklist"]
 >
-> - **Step 1: Provision a database in Azure SQL Database**. Contoso provisions an Azure SQL Database instance. After the application website is migrated to Azure, the WCF service web app will point to this instance.
-> - **Step 2: Assess the database**. Contoso assesses the database for migration by using Data Migration Assistant and then migrates it via Azure Database Migration Service.
-> - **Step 3: Provision web apps**. Contoso provisions the two web apps.
+> - **Step 1: Assess and migrate the web apps.**. Contoso uses the [Azure App Service Migration Assistant](https://azure.microsoft.com/en-us/migration/web-applications/) tool to run pre-migration compatibility checks and migrate their web apps to Azure App Service.
```
Comment (JanetCThomas) [inline docs/migrate/azure-best-practices/contoso-migration-refactor-web-app-sql.md:111]: please remove en-us from the URL.

Comment (JanetCThomas) [reply docs/migrate/azure-best-practices/contoso-migration-refactor-web-app-sql.md:111]: URL fixed in PR 284

**Comment by PRMerger20** [comment]
@JanetCThomas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by JanetCThomas** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@JanetCThomas: I'm sorry - only the author of this article, @BrianBlanchard, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger19** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**